confero/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['confero-env.cvqp3jgbbb.us-east-2.elasticbeanstalk.com']
```

Comment:
Hopefully we'll be able to put this into an environment variable at some point, but works for now :+1: